pkg/sidecar/config/config.go

```diff
@@ -296,6 +301,15 @@ func (s *ScyllaConfig) validateCpuSet(ctx context.Context, cpusAllowed string, s
 	return nil
 }
 
+func (s *ScyllaConfig) setupIOTuneCache() error {
+	if _, err := os.Stat(scyllaIOPropertiesPath); errors.Is(err, os.ErrNotExist) {
+		if err := os.Symlink(path.Join(naming.DataDir, naming.ScyllaIOPropertiesName), scyllaIOPropertiesPath); err != nil {
```

Comment:
nit: pls split the variable assignment, the lines are already long

Reply:
done